tests/integration/check-missing-files

```diff
@@ -24,6 +24,7 @@ for xml in "$@"; do
           abl.elf) ;;
           aop.mbn) ;;
           boot.img) ;;
+          cdt.bin) ;;
```

Comment:
Missing description of the problem in the commit message

Reply:
and it should come before the patch that includes --cdt.